cryosparc/job.py

```diff
@@ -219,6 +219,27 @@ def wait_for_done(self, error_on_incomplete: bool = False, timeout: Optional[int
         ), f"Job {self.project_uid}-{self.uid} did not complete (status {status})"
         return status
 
+    def interact(self, action: str, body: Any = {}, timeout: int = 10, refresh: bool = False) -> Any:
+        """
+        Call an interactive action on a waiting interactive job. The possible
+        actions and expected body depends on the job type.
+
+        Args:
+            action (str): Interactive endpoint to call.
+            body (any): Body parameters for the interactive endpoint. Must be
+                JSON-encodable.
+            timeout (int, optional): Maximum time to wait for the action to
+                complete, in seconds. Defaults to 10.
+            refresh (bool, optional): If True, defresh the job document after
+                posting. Defaults to False.
```

Comment:
why not default to True?

Reply:
It's not guaranteed that the job document will change after interacting with it. It depends on what interactive function gets called.